packages/govuk-frontend/src/govuk/components/accordion/accordion.mjs

```diff
@@ -394,18 +410,30 @@ export class Accordion extends GOVUKFrontendComponent {
     const $showHideText = $section.querySelector(
       `.${this.sectionShowHideTextClass}`
     )
+
     const $button = $section.querySelector(`.${this.sectionButtonClass}`)
     const $content = $section.querySelector(`.${this.sectionContentClass}`)
 
-    if (
-      !$showHideIcon ||
-      !($showHideText instanceof HTMLElement) ||
-      !$button ||
-      !$content
-    ) {
+    // Return early for elements created during set up
+    if (!$showHideIcon || !$showHideText) {
       return
     }
 
+    if (!$button || !$content) {
```

Comment:
What do think about merging all these into one?

```suggestion
    if (!$showHideIcon || $showHideText || !$button || !$content) {
```

Reply:
I thought we'd made the call to silently return for the first two? Do you mean have 'em all in one check, then only throw for the relevant ones?

Reply:
Ah, right! Sorry, didn't click that the comments were connected. It makes sense now, thanks!